src/community/china/index.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Using Flutter in China
+description: Where to find a version of the Flutter site that is localized to Simplified Chinese.
+toc: true
+os-list: [Windows, macOS, Linux, ChromeOS]
+---
+
+{% assign path = 'flutter_infra_release/releases/stable/windows/flutter_windows_3.3.0-stable.zip' -%}
+
+To speed the download and installation of Flutter in China,
+consider using a mirror site.
+
+{{site.alert.important}}
+  Use mirror sites only if you _trust_ the provider.
+  The Flutter team cannot verify their reliability or security.
+{{site.alert.end}}
+
+## Use a Flutter mirror site
+
+The [China Flutter User Group][] (CFUG) maintains a Simplified Chinese
+Flutter website [https://flutter.cn](https://flutter.cn) and a mirror.
+Other mirrors can be found at the [end of this guide](#known-trusted-community-run-mirror-sites).
```

Comment:
I think it's good to stick to the full "mirror sites" wording to be consistent and avoid any potential confusion.

```suggestion
Other mirror sites can be found at the [end of this guide](#known-trusted-community-run-mirror-sites).
```

Reply:
Added a link to "mirror site" and referenced the two together.